src/views/AccountDetail.vue

```diff
@@ -11,7 +11,7 @@
         <template #error>Account {{address}} is not exist</template>
       </Card>
 
-      <Card v-if="showMultiSigDeatail" class="card-f card-adaptive" :loading="loading">
+      <Card v-if="1||showMultiSigDeatail" class="card-f card-adaptive" :loading="loading">
```

Comment:
what is the `1` meaning?

Reply:
Thanks. Missed to delete it. I use this as a "comment" for "if".